horace_core/sqw/file_io/@sqw_binfile_common/sqw_binfile_common.m

```diff
@@ -195,7 +195,11 @@
         
         
         % read pixels information
-        pix    = get_pix(obj,varargin);
+        pix = get_pix(obj,varargin);
+        % read pixels at the given indices
+        pix = get_pix_at_indices(obj,indices);
```

Comment:
What is the point of overwriting pix 3 times here? Unless I'm missing something

Reply:
this^^^

Reply:
These are method declarations